deeplearning4j-zoo/src/main/java/org/deeplearning4j/zoo/model/AlexNet.java

```diff
@@ -69,66 +61,104 @@ public long pretrainedChecksum(PretrainedType pretrainedType) {
         return 0L;
     }
 
-    @Override
-    public ZooType zooType() {
-        return ZooType.ALEXNET;
-    }
-
     @Override
     public Class<? extends Model> modelType() {
         return MultiLayerNetwork.class;
     }
 
     public MultiLayerConfiguration conf() {
         double nonZeroBias = 1;
-        double dropOut = 0.5;
 
         MultiLayerConfiguration conf = new NeuralNetConfiguration.Builder().seed(seed)
-                        .weightInit(WeightInit.DISTRIBUTION).dist(new NormalDistribution(0.0, 0.01))
-                        .activation(Activation.RELU).optimizationAlgo(OptimizationAlgorithm.STOCHASTIC_GRADIENT_DESCENT)
-                        .updater(new Nesterovs(1e-2, 0.9))
+                        .weightInit(WeightInit.DISTRIBUTION)
+                        .dist(new NormalDistribution(0.0, 0.01))
```

Comment:
Not a big deal - but .weightInit(Distribution) is a thing - equivalent to the 2 .weightInit(DISTRIBUTION).dist(y) calls.